### ISSUE TITLE
Issue with MongoDb connection in the FastReport Community app

### ISSUE THREAD

**Issue by echaritonidis** [FastReports/FastReport#203]
Hello,

There seems to be an issue with the app when I want to add a mongo db data source.
I'm posting the exception log

```
See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.FileLoadException: Could not load file or assembly 'MongoDB.Driver, Version=2.3.0.157, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
File name: 'MongoDB.Driver, Version=2.3.0.157, Culture=neutral, PublicKeyToken=null'
   at FastReport.Data.MongoDBConnectionEditor.SetConnectionString(String value)
   at FastReport.Forms.ConnectionForm.cbxConnections_SelectedIndexChanged(Object sender, EventArgs e)
   at System.Windows.Forms.ComboBox.OnSelectedIndexChanged(EventArgs e)
   at System.Windows.Forms.ComboBox.set_SelectedIndex(Int32 value)
   at FastReport.Forms.ConnectionForm.EnumConnections()
   at FastReport.Forms.ConnectionForm..ctor()
   at FastReport.Forms.DataWizardForm.btnNewConnection_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)




************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4200.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
Designer
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4200.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4200.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4200.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
FastReport
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Bars
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Editor
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
FastReport.DataVisualization
    Assembly Version: 2020.3.1.0
    Win32 Version: 2020.3.1.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/FastReport.DataVisualization.DLL
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4200.0 built by: NET48REL1LAST_C
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------
FastReport.Compat
    Assembly Version: 2020.3.2.0
    Win32 Version: 2020.3.2.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/FastReport.Compat.DLL
----------------------------------------
FastReport.Data.Couchbase
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Data.Json
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/FastReport.Data.Json.DLL
----------------------------------------
FastReport.Data.Json
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
Newtonsoft.Json
    Assembly Version: 11.0.0.0
    Win32 Version: 11.0.2.21924
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Newtonsoft.Json.DLL
----------------------------------------
FastReport.Data.MongoDB
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Data.MySql
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Data.OracleODPCore
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Data.Postgres
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Data.RavenDB
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.Data.SQLite
    Assembly Version: 2020.3.0.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
FastReport.DataVisualization
    Assembly Version: 2020.3.1.0
    Win32 Version: 2020.3.0.0
    CodeBase: file:///C:/Users/e.charitonidis/Desktop/FastReport.Community/Designer.exe
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4190.0 built by: NET48REL1LAST_B
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.8.4084.0 built by: NET48REL1
    CodeBase: file:///C:/WINDOWS/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.
```

**Comment by Detrav** [comment]
Hi there is a dll signing problem on mongo libraries. This seems to be a problem after updating .net 4.8 to the final version.

I will create a task for the team, they will update the mongo version for the next release.

**Comment by echaritonidis** [comment]
Thanks for checking up! please let me know when it's fixed.

**Comment by KirillKornienko** [comment]
Hello @echaritonidis ! We fixed this error in the last update 2020.4.0. Please, try this changes and share your opinion.

**Comment by echaritonidis** [comment]
Hi @KirillKornienko, thanks for letting me know!
